### PR TITLE
Update to chemfiles 0.8

### PR DESCRIPTION
**Pull request by Luthaf**
Do not merge until the 0.8.0 version is actually released!

### CODE REVIEW

**Comment by Luthaf** [comment]
So re-rendering the recipe with conda-smithy 2.4.4 removed Python3 windows builders (https://github.com/conda-forge/chemfiles-lib-feedstock/pull/11/commits/babd3ca03be70ddb2a02aa23dd75afec6dd1fd39#diff-180360612c6b8c4ed830919bbb4dd459L20). @jakirkham, any idea why? These are the one I need =)

**Comment by jakirkham** [comment]
Not sure. Let's try the bot.

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,10 +30,10 @@ test:
 
 about:
   home: https://chemfiles.org
-  license: MPL-v2.0
-  license_file: LICENCE.txt
+  license: BSD
```
Comment (jakirkham) [inline recipe/meta.yaml]: Is this `2-Clause` or `3-Clause`?

Comment (Luthaf) [reply recipe/meta.yaml]: 3-clause. I tried to look at how one should specify licenses, but I could not find any definitive information. Should I put it as `BSD-3-Clauses` ?

**Comment by jakirkham** [comment]
I think it is because you need `python` in `build`. Otherwise the feature stuff doesn't work correctly. This is only needed for Windows.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Typically we do `BSD 3-Clause`, but an extra `-` is probably fine. We aren't too picky about it yet.

**Comment by Luthaf** [comment]
> I think it is because you need python in build. Otherwise the feature stuff doesn't work correctly. This is only needed for Windows.

Yes, it is it! Thank you very much for the quick answer 😄

File: recipe/meta.yaml (hunk 1 above)
Comment (Luthaf) [reply recipe/meta.yaml]: The extra dash comes from https://spdx.org/licenses/ identifiers, which is what cargo (rust package manager) uses.

**Comment by jakirkham** [comment]
np. Glad things are working now. :)

**Pull request merged**